job_templates/packaging_job.xml.em

```diff
@@ -238,10 +238,15 @@ rmdir /S /Q ws workspace
 
 set CONTAINER_NAME=ros2_windows_ci_%CI_ROS_DISTRO%
 set DOCKERFILE=windows_docker_resources\Dockerfile
+set SOLOFILE=windows_docker_resources\install_ros2_%CI_ROS_DISTRO%.json
```

Comment:
```suggestion
set SOLO_FILE=windows_docker_resources\install_ros2_%CI_ROS_DISTRO%.json
```

Companion to an earlier suggestion.

Reply:
Fixed in [b7ae4a5](https://github.com/ros2/ci/pull/786/commits/b7ae4a59d079670a8f7a8772d5ba0b3102013732)